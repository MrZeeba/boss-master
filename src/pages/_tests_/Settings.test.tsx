import React from 'react';
import renderer from 'react-test-renderer';

import Settings from '../Settings';

describe('<View />', () => {
  it('has 1 children', () => {
    const tree = renderer.create(<Settings />).toJSON();
    expect(tree.children.length).toBe(1);
  });
});

it('renders correctly', () => {
  const tree = renderer.create(<Settings />).toJSON();
  expect(tree).toMatchSnapshot();
});
