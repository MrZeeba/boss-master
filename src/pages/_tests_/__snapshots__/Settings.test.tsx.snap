// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <Text>
    This is the settings page
  </Text>
</View>
`;
